<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Only change those fields if you know what to do!</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>10</y>
     <width>340</width>
     <height>241</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>340</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="General">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>80</y>
       <width>321</width>
       <height>121</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>330</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Offline data</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>161</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Path to offline data:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="offlineDataField">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="offlineDataFolderBrowse">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>80</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse...</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="offlineData_checkBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Use offline data</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>251</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Default path to the .epw output folder</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="outputPathField">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="outputhPathFolderBrowse">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>40</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse...</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Advanced</string>
    </attribute>
    <widget class="QLabel" name="label_4">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>10</y>
       <width>321</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Only change those fields if you know what to do!</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>offlineDataFolderBrowse</sender>
   <signal>clicked()</signal>
   <receiver>Settings</receiver>
   <slot>offlineDataFolderBrowse_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>81</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>outputhPathFolderBrowse</sender>
   <signal>clicked()</signal>
   <receiver>Settings</receiver>
   <slot>outputPathFieldBrowse_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>offlineDataFolderBrowse_clicked()</slot>
  <slot>outputPathFieldBrowse_clicked()</slot>
 </slots>
</ui>
